samples/apps/copilot-chat-app/webapi/Skills/ChatSkill.cs

```diff
@@ -94,15 +104,29 @@ public async Task<string> ExtractUserMemoriesAsync(SKContext context)
         // Find the most recent message.
         var latestMessage = await this._chatMessageRepository.FindLastByChatIdAsync(chatId);
 
+        // Search for relevant memories.
+        List<MemoryQueryResult> relevantMemories = new List<MemoryQueryResult>();
+        foreach (var memoryName in SystemPromptDefaults.MemoryMap.Keys)
+        {
+            var results = context.Memory.SearchAsync(
+                SemanticMemoryExtractor.MemoryCollectionName(chatId, memoryName),
+                latestMessage.ToString(),
+                limit: 1000,
+                minRelevanceScore: 0.8);
```

Comment:
Should that 0.8 be (eventually, not necessarily today) configurable?

Reply:
Yes, eventually, I think we are thinking about moving the prompt configurations to a config file too. We can make this param configurable when we do that. 

We should be fine for now.